bundle/ScheduledVisibility/Handler/ContentAndLocation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler;
+
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\Repository\Repository;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\ScheduledVisibilityInterface;
+
+final class ContentAndLocation implements ScheduledVisibilityInterface
+{
+    public function __construct(
+        private readonly Repository $repository,
+        private readonly ContentService $contentService,
+        private readonly LocationService $locationService,
+    ) {}
+
+    public function hide(Content $content): void
+    {
+        $this->repository->sudo(
```

Comment:
This can be combined into a single `sudo()` call (on other places as well).

Reply:
fixed in 091f0e24f3b9c40dd141508041b8403dee4c8b21